src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -1,25 +1,26 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  
   <PropertyGroup>
-    <Version>4.1.0</Version>
+    <Version>4.1.1</Version>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
   </PropertyGroup>
-
   <ItemGroup>
-  <RuntimeDependency Include="runtime.win10-amd64-aot.runtime.native.System.IO.Compression">
+    <RuntimeDependency Include="runtime.win10-x64-aot.runtime.native.System.IO.Compression">
```

Comment:
No looks like we did fix it in the release branch. 

---
In reply to: [67385318](https://github.com/dotnet/corefx/pull/9455#discussion_r67385318) [](ancestors = 67385318)